sequencer/batchbuilder.go

```diff
@@ -73,23 +77,18 @@ func (s *Sequencer) tryToProcessTx(ctx context.Context, ticker *time.Ticker) {
 		return
 	}
 	// process batch
-	log.Infof("processing batch with %d txs. %d txs are new from this iteration", len(s.sequenceInProgress.Txs), appendedTxsAmount)
+	log.Infof("processing tx number [%d] for current batch...", len(s.sequenceInProgress.Txs))
```

Comment:
```suggestion
	log.Infof("processing [%d] transactions for current batch...", len(s.sequenceInProgress.Txs))
```
@Psykepro WDYT of changing the sentence like suggested? IMO `tx number` sounds like it's processing a single transaction with ID or index `[%d]`.

Same for the other logs.

Reply:
This is misleading because here we are processing a specific transaction for the batch otherwise it will look like we are processing 3 transactions instead of transaction number 3 when we are processing 1 by 1 for example.

Reply:
@Psykepro ok then I probably misunderstood the meaning of the log. It really tells about the index of the current processed transaction, right. The index is always the latest transaction in `s.sequenceInProgress.Txs`, that's why you use `len`, isnt' it?

Reply:
Yep, that's true! :))